packages/core/src/browser/style/tabs.css

```diff
@@ -522,25 +523,13 @@
   background: var(--theia-icon-close) no-repeat;
 }
 
-/** Configure layout of a toolbar item that shows a pop-up menu. */
-.p-TabBar-toolbar .item.menu {
-  display: grid;
-}
-
-/** The elements of the item that shows a pop-up menu are stack atop one other. */
-.p-TabBar-toolbar .item.menu > div {
-  grid-area: 1 / 1;
-}
-
 /**
  * The chevron for the pop-up menu indication is shrunk and
  * stuffed in the bottom-right corner.
  */
-.p-TabBar-toolbar .item.menu > .chevron {
-  scale: 50%;
-  align-self: end;
-  justify-self: end;
-  translate: 5px 3px;
+.p-TabBar-toolbar .item.menu .chevron {
+  font-size: 8px;
+  vertical-align: bottom;
```

Comment:
I was not trying to imitate VS Code visually, but to keep the "chevron at the bottom" look we had before while implementing the functionality of having both a command and a drop-down. Do you think the VS Code version looks better?

Reply:
I think looks better - also I was confused by the small size of the chevron - makes it a bit difficult to click on it, since the whole button is just a few pixel in widget.